Assets/Scripts/Augment/BulletModifiers/SpiralPathModifier.cs

```diff
@@ -35,10 +44,9 @@ public void addSpiralDisplacement(float distance, ref ProjectileState state, Gun
         state.position += state.rotation * (newVector - oldVector);
  
     }
-    private void Start()
+
+    public Priority GetPriority()
```

Comment:
It could be a private property, but I think it's a bit overkill to force all modifiers inherit from a class when that would be the only field. Especially given the reason for using priority I gave above.